python/dlisio/plumbing/parameter.py

```diff
@@ -137,10 +137,17 @@ def values(self):
     def describe_attr(self, buf, width, indent, exclude):
         describe_description(buf, self.long_name, width, indent, exclude)
 
+        units = ''
+        try:
+            units = self.attic['VALUES'].units
+        except:
+            pass
+
         d = OrderedDict()
         d['Sample dimensions'] = self.dimension
         d['Axis labels']       = self.axis
         d['Zones']             = self.zones
+        d['Units']             = units
```

Comment:
I admit I saw it somewhat differently when I suggested it. In my mind outcome must be something like:

```
  Calibration-measurement
   -----------------------
   name   : MSCT/KPPO_PM/CALI_MEASUREMENT
   origin : 2
   copy   : 0

   Metadata
   --
   Phase in job sequence     : BEFORE
   Start time of acquisition : 2011-08-20 18:27:54
   Duration time             : 5 s

   Samples
   --
   Number of values                    : 1
   Samples used to compute std/max-dev : 12

   Value(s)      : 2.5799591541290283 in
   Reference     : 2.140000104904175 in
   Std deviation : 0.004975703079253435 in
   Max deviation : 0.010652303695678711 in
```
or

```
   ---------
   Equipment
   ---------
   name   : MSCT/MCFU_1/EQUIPMENT
   origin : 2
   copy   : 0

   Trademark name     : MCFU_1-AA
   Operational status : 1
   Serial number      : 119.

   Length         : 125.0 in
   Diameter (min) : 5.25 in
   Volume         : 1.5700000524520874 ft3
   Weight         : 144.0 lbm

   Max pressure          : 20000.0 psi
   Max temperature       : 350.0 degF
```

Not sure if that's the same issue you are wondering about if it's worth it, because I probably think it is.

It's not very fast to have it, but doesn't seem like too much code to me. Roughly updated 
- lines in objects to something like `d['Type of measurement']       = "TYPE"`
- do `describe_attributes(buf, d, self.attic, width, indent, exclude) `instead of dict
- updated `describe_array` and `describe_text` signatures to include units and actually print them
- changed `describe_dict` to `describe_tup_dict` (so value is (value, units))
- `describe_sampled_attrs` and `describe_attributes` retrieve units as they know attic and label

Obv, I am missing a lot of useful things, but still.

Besides, even if units were take from a proper place, we would still have to deal with it. So maybe now is better than later.

(The biggest change is all the labels. Maybe we should again think about 1-1 mapping between our attribute names and attic labels. Maybe we can do something about it so we always use self["LABEL"] inside of the code and leave self.label barely for users benefit)

Reply:
I'm gonna bother with this now, as I do not think it's worth the effort.